cylc/flow/tui/app.py

```diff
@@ -206,16 +246,25 @@ class TuiApp:
 
     """
 
-    UPDATE_INTERVAL = 1
-    CLIENT_TIMEOUT = 1
+    # the UI update interval
+    # NOTE: this is different from the data update interval
+    UPDATE_INTERVAL = 0.1
```

Comment:
This means that the UI will look for updates every 0.1 seconds, but the frequency of these updates is determined by the updater which is decoupled and runs to its own clock.